src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <>
+      <h1>Hi from app</h1>
+
+      <Display />
```

Comment:
- You don't need `<h1>Hi from app</h1>` so remove it. Wrap the `<Display />` component in the `<div>` component.
```html
<div>
    <Display />
    <ButtonPanel />
</div>
```